.gitignore

```diff
@@ -63,4 +63,5 @@ git push origin master
 /.vs
 *.dat
 /StateTemplate.sln
-/website.publishproj
\ No newline at end of file
+/website.publishproj
+src/.DS_Store
```

Comment:
There are Git commands at the end of the file that might be making gitignore work incorrectly.